python/pfs/drp/stella/createDetGeom.py

```diff
@@ -0,0 +1,108 @@
+import os
+import sys
+import argparse
+
+import lsst.pex.config                  as pexConfig
+import lsst.afw.table as afwTable
+import lsst.afw.geom as afwGeom
+from lsst.pipe.base import Task
+
+class CreateDetGeomConfig(pexConfig.Config):
+    """Configuration for creating the PFS detector geometry files"""
+
+class CreateDetGeomTask(Task):
+    """Task to create the detector geometry files for PFS"""
+    ConfigClass = CreateDetGeomConfig
+    _DefaultName = "createDetGeomTask"
+
+    def __init__(self, *args, **kwargs):
+        super(CreateDetGeomTask, self).__init__(*args, **kwargs)
+
+    def run(self):
+        gain = (1.24, 1.24, 1.27, 1.18, 1.26, 1.20, 1.24, 1.26)
+        rdnoise = (3.61, 3.78, 3.18, 2.95, 3.19, 3.80, 4.51, 3.18)
+        
+        outDir = os.getenv('OBS_PFS_DIR', '~/stella_git/obs_pfs')
+        print 'outDir = <',outDir,'>'
+        outDir = outDir+'/pfs/camera'
+        
+        amp = 0
+        for iArm in range(3):
+            for iCCD in range(4):
+                detector = afwTable.AmpInfoCatalog(afwTable.AmpInfoTable.makeMinimalSchema())
+
+                for iAmp in range(8):
+
+                    x0 = iAmp * 552
+                    x1 = ((iAmp + 1) * 552)-1
```

Comment:
There's a lot of numbers hard-coded in here. Is that necessary or desirable? Could they change in future? Should they be configurable, or read from some separate data file, rather than directly embedded in Python? Can you provide references to the source material?